roles/gateway_export_database/README.md

```diff
@@ -5,6 +5,15 @@ This role will export Gateway and OTK database from source gateway and store in
 
 Requirements
 ------------
+* Purge audit events manually from the source Gateway. For more information, see "Delete Audit Events" in View Gateway Audit Events in tech docs. 
```

Comment:
Might wanna just link to [Delete Audit Events](https://techdocs.broadcom.com/content/broadcom/techdocs/us/en/ca-enterprise-software/layer7-api-management/api-gateway/10-0/security-configuration-in-policy-manager/tasks-menu-security-options/view-gateway-audit-events.html) in TechDocs instead of repeating the steps here

Reply:
It's bury in the tech doc and took me a while to find it. I'm trying to save customer some time